starter/js/atm.js

```diff
@@ -1,42 +1,105 @@
 //Begin with the document ready function
+$(document).ready(function() {
+      //On click of the depositChecking button
+      $('#depositChecking').on('click', checkingDepo);
 
-    //Checking account deposit function
+      //On click of the withdrawChecking button
+      $('#withdrawChecking').on('click', checkingWithdraw);
 
-      //On click of the depositChecking button
+      //On click of the depositSavings button
+      $('#depositSavings').on('click', savingsDepo);
+
+      //On click of the withdrawl savings button
+      $('#withdrawSavings').on('click', savingsWithdraw);
+});
+//Checking account deposit function
+function checkingDepo() {
+  console.log('checking depo');
+
+  //Get value from the amountChecking input field
+  var toAdd = $('#amountChecking').val();
+
+  if (toAdd < 0) {
+    alert('you silly goose, you cant deposit a negative number');
+  } else {
+
+  var currentBal = parseInt($('#checkingBalance').text());
+  console.log(currentBal);
+  var newBal = currentBal + parseInt(toAdd);
+  console.log('newbalance',newBal);
+
+  $('#checkingBalance').text(newBal);
+  }
+}
+//Checking account withdrawl funtion
+function checkingWithdraw(){
+  console.log('checking withdraw');
+  //Get value from the amountChecking input field
+  var toTake = $('#amountChecking').val();
+  var currentBal = parseInt($('#checkingBalance').text());
+  // If that value is greater than the value in the account ignore that action
+  // In other words if this would put the account into a negative balance do not allow it
+  if (parseInt(toTake) > currentBal) {
+    return;
+  } else {  //Else subtract that value from the current amount in the checking account
+    currentBal -= parseInt(toTake);
```

Comment:
Since you are repeating `parseInt(toTake)`, is there something you could have done to reduce that repetition? Maybe make that a variable altogether? Or reset the variable toTake somewhere?